samples/modules/lvgl/prj.conf

```diff
@@ -0,0 +1,17 @@
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_LOG=y
+CONFIG_SHELL=y
```

Comment:
Not sure if the shell has an added value in this sample.

Reply:
Thought it might be useful to have the `lvgl stats memory` command available since the purpose of the stress demo is also to exercise the memory management (and check for leaks).